r/tools/nixlibs.R

```diff
@@ -215,7 +215,7 @@ determine_binary_from_stderr <- function(errs) {
     # If we're here, we're on an older OS but with a newer compiler than gcc 4.8
     # (we already checked), so it is possible to build with more features on.
     # We just can't use our binaries because they were built with newer glibc.
-    return(NULL)
+    return("centos-7")
```

Comment:
If we're doing this, can we delete the ubuntu-18.04 job and use the centos-7 one for those cases?

Reply:
possibly? I don't really know but I will open a jira for it

Reply:
https://issues.apache.org/jira/browse/ARROW-17876

Reply:
Duplicate of ARROW-16976?